io.github.ruffsl.ctrlassist.yml

```diff
@@ -0,0 +1,33 @@
+app-id: io.github.ruffsl.ctrlassist
+icon: io.github.ruffsl.ctrlassist
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: ctrlassist
+finish-args:
```

Comment:
so how is this gonna display something?

Reply:
CLI/TUI are not allowable on Flathub.

Nor "applet". What happen on, say, GNOME?

Reply:
See https://docs.flathub.org/docs/for-app-authors/requirements#impermissible-submissions